### PR TITLE
Sponsored users

### LINKED ISSUE #304: Vo - settings - sponsored users
Add new page for showing sponsored users. The page should contain table where sponsored users would be listed. New columns should be here for who is sponsoring the user. Then button for adding new sponsored user and then for removing the sponsored user. Page also should contain reload button and filter field.
This functionality in mostly not implemented in old gui. You can only find it in the Vo overview in quick tools: "Create service member". After opening it there is option "user type" where user can choose what he want to create. Since we want only create sponsored user that row and options shouldnt be here.
This is kind of new feature so ask for more information Slávek or me.

### PR DESCRIPTION
**Pull request by bodnara**
Resolves #304 

Sponsored members

* added page with sponsored members to Vo settings

### CODE REVIEW

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    Urns.MEMBER_DEF_MAIL,
+    Urns.MEMBER_DEF_EXPIRATION
+  ];
+
+  selection = new SelectionModel<MemberWithSponsors>(true, []);
+  searchString = '';
+  loading = false;
+  statuses = new FormControl();
+  statusList = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
+  selectedStatuses: string[] = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts:47]: I dont understand why it is here

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  pageSize: number;
+  tableId = TABLE_VO_MEMBERS;
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts:49]: is this tableId correct?

**Review by xkureck — CHANGES_REQUESTED** [review]
image
filter table by name, not ID
in the table when clicking on some member, we would like him to reroute to the member detail
in the sponsors column we would like to have here the names of the sponsors
Then the button that opens dialog "Edit member sponsors" - The icon for that button I would actually like the three dots(more_horiz) and move it to another new column(no label to to column header).

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts (hunk 1 above)
Comment (bodnara) [reply apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts]: This was used when this page was using member list, now it was useless so I deleted it.

**Review by xkureck — APPROVED** [review]

**Pull request merged**